src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,37 @@
+.stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+
+  &__wrap {
+    display: flex;
+
+    &--4 :nth-child(-n + 4) {
+      background-image: url(../images/star-active.svg);
+    }
+  }
+
+  &__star {
+    background-image: url(../images/star.svg);
+    width: 16px;
+    height: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
+
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  &__reviews {
+    margin: 0;
+    font-size: 10px;
```

Comment:
Stars block shouldn't know anything about review, just render stars

Reply:
why it was ok from the previous task Product card?

Reply:
> Stars block shouldn't know anything about review, just render stars

Not fixed